### PR TITLE
add teradata-bigquery-sql-translator

### PR DESCRIPTION
**Pull request by jasonmar**
Added BigQuery Standard SQL Translation utility

### CODE REVIEW

File: tools/teradata-bigquery-sql-translator/.gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+!project/plugins.sbt
+*.DS_Store
+*.gz
+*.iml
+*.ipr
+*.iws
+*.log
+*.swp
+.idea
```
Comment (zhaber) [inline tools/teradata-bigquery-sql-translator/.gitignore]: do you need both .idea and .idea/*? won't just .idea/ work? Same for target/*

Comment (jasonmar) [reply tools/teradata-bigquery-sql-translator/.gitignore]: You're right, they're not necessary. It was left over from the original project. Thanks for catching this, I realized that project was being excluded and I've added build.properties and plugins.sbt to the PR.

File: tools/teradata-bigquery-sql-translator/src/main/scala/com/google/cloud/pso/sql/SQLFmt.scala [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  def ~/(o: SQLFmt): SQLFmt = Group(this :: NewLine :: o :: Nil)
+
+  def ~/(o: Option[SQLFmt]): SQLFmt = o.map(o => Group(this :: NewLine :: o :: Nil)).getOrElse(this)
+
+  def ~|(o: SQLFmt*) = Group(this :: Indented(Group(o.toList)) :: Nil)
+
+  def ~|(o: Option[SQLFmt]): SQLFmt = o.map(o => Group(this :: Indented(Group(List(o))) :: Nil)).getOrElse(this)
+
+  def print(style: Fmt = Fmt.Default): String =
+    SQLFmt.toSQL(this, style).right.getOrElse(sys.error("Failed to print SQL"))
```
Comment (zhaber) [inline tools/teradata-bigquery-sql-translator/src/main/scala/com/google/cloud/pso/sql/SQLFmt.scala]: Would including an error message from Either.left into sys.error be helpful?

File: tools/teradata-bigquery-sql-translator/README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- LOCK TABLE
+- DIAGNOSTIC
+- DROP TABLE
+
+
+## Usage
+
+There are two executable classes in this project:
+1. RewriteDir - rewrite all SQL files in a directory
+2. SQLServlet - local webserver for SQL translation
```
Comment (zhaber) [inline tools/teradata-bigquery-sql-translator/README.md]: That's really nice

File: tools/teradata-bigquery-sql-translator/LICENSE [hunk 4]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (ryanmcdowell) [inline tools/teradata-bigquery-sql-translator/LICENSE]: The LICENSE file isn't needed here as the subdirectory will be covered by the root license.

File: tools/teradata-bigquery-sql-translator/src/main/resources/static/style.css [hunk 5]
```diff
@@ -0,0 +1,172 @@
+html,body,input{
```
Comment (ryanmcdowell) [inline tools/teradata-bigquery-sql-translator/src/main/resources/static/style.css]: License header?

File: tools/teradata-bigquery-sql-translator/README.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+#### Run from Jar
+```
+sbt assembly
+java -jar target/scala-2.11/sql-assembly-0.1.0-SNAPSHOT.jar /path/to/sql
+```
+
+#### Checking in the files
+
+Typically the directory being rewritten should be within a git repository. To check in the modified files after running the application:
```
Comment (zhaber) [inline tools/teradata-bigquery-sql-translator/README.md]: Should we check that there is .git directory

File: tools/teradata-bigquery-sql-translator/README.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+Parsing failed for sql/statement2.sql: [*] Parsing error: ; expected at 233
+
+A.DATE1, A.DATE2) as X,
+OTRANSLATE[*](A.X, ' -./\()~!@#$%^&*-_=+\|][}{,./<>?;:','') AS Y
+FROM B
+
+
+Success:	1234
+Failure:	123
```
Comment (zhaber) [inline tools/teradata-bigquery-sql-translator/README.md]: Is the list of files stored anywhere?

**Comment by jaketf** [comment]
@jasonmar My understanding is we're waiting to merge this from some sign-off from Chad. Please address the Feb 19 feedback so we're ready to go. I'll remove the stale tag for now.

**Pull request closed without merging**